### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by yohannesdagnachew**
- Description
- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- Build your project schema.
  - Create and run the necessary migration files.
  - Table and column names should match the ERD diagram.
  - Foreign keys should be included.
  - All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

**Review by julie-ify — COMMENTED** [review]
Hi @yohannesdagnachew,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Your rubocop linter has failed

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/migrate/20220920153113_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.integer :like_counter
+      t.integer :comment_counter
+
+      t.timestamps
+    end
+  end
+end
```
Comment (m05e5) [inline db/migrate/20220920153113_create_posts.rb]: - [ ] Hi. As shown in your ERD diagram, the Post table has a property text so kindly add it to your migrations
image

**Review by m05e5 — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @yohannesdagnachew  , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] _Please see comments under this review_
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_m05e5_


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220920153113_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
```
Comment (DammyShittu) [inline db/migrate/20220920153113_create_posts.rb:5]: - [ ] Merely adding this text here does not add the `text` property to your Post table. You need to generate another migration file for adding a column to the posts table and run `db:migrate` again. If you look at your schema.rb file you would see that the text property is not in the posts table.

  You can take a look at this answer on how to generate migrations in rails: [https://stackoverflow.com/a/4834906](https://stackoverflow.com/a/4834906)

File: README.md [hunk 3]
```diff
@@ -1 +1,24 @@
-# Blog-app
\ No newline at end of file
+# README
```
Comment (DammyShittu) [inline README.md]: - **[OPTIONAL]** I would recommend you edit your readme docs using the microverse template

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @yohannesdagnachew,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @yohannesdagnachew,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**